tools/ld-analyse/CMakeLists.txt

```diff
@@ -0,0 +1,246 @@
+message( STATUS "---------Beginning cmake build of ld-analyse---------")
+
```

Comment:
The subdirs all check for dependencies and the Git version individually, so there's a lot of duplication. It would be better to check for all the deps in the top-level `CMakeLists.txt` (as GNURadio etc. do) - then you can also enable/disable subdirs if dependencies are missing.